sdks/python/apache_beam/runners/dataflow_runner.py

```diff
@@ -180,9 +175,6 @@ def run(self, pipeline):
     self.result = DataflowPipelineResult(
         self.dataflow_client.create_job(self.job), self)
 
-    if self.result.has_job and self.blocking:
-      self.result.wait_until_finish()
-
     return self.result
```

Comment:
Do we need `self.result` any more? Can we just return the `DataflowPipelineResult` object?